steps/evaluation.py

```diff
@@ -51,6 +57,12 @@ def get_expectation_values_for_qubit_operator(
     save_expectation_values(expectation_values, "expectation-values.json")
 
 
+def get_ground_state_rdm_from_qubit_operator(qubit_operator, n_particles):
```

Comment:
Please add a docstring and specify the types of input parameters.

Reply:
Addressed in 0ff7e3c086d6027c44ee79a7c917c44d3a7fdb48